bin/cli.js

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env node
+
+const { logger, kebabcasify, parseContent } = require('../utils')
+const kleur = require('kleur')
+const path = require('path')
+const fs = require('fs-extra')
+const readline = require('readline-sync')
+const glob = require('glob')
+const templates = glob.sync(path.join(__dirname, '../templates/**'), {
+  nodir: true,
+  dot: true
+})
+
+/**
+ * Prompt user for input to populate template files
+ */
+const npmName = readline.question(
+  '✍️  What is the npm name of your component? '
+)
+
+const componentName = kebabcasify(npmName)
+const outDir = path.join(process.cwd(), componentName)
+
+/**
+ * Created template files
+ */
+templates.forEach(filepath => {
```

Comment:
let this become a fucntion, so you just need a meaningful function, not a comment